src/cz/zipek/sqflint/linter/PreProcessorError.java

```diff
@@ -0,0 +1,64 @@
+package cz.zipek.sqflint.linter;
+
+import java.io.Serializable;
+
+import cz.zipek.sqflint.parser.Token;
+
+/**
+ *
+ * @author Senfo
+ */
+public class PreProcessorError implements Serializable {
+	
+	/**
+	 * serial
+	 */
+	private static final long serialVersionUID = 1L;
```

Comment:
What is this used for?

Reply:
It inherits a class which implements serializable (dunno why)
Every serializable class needs this variable, this is just to make sure it has one (the parent also dont specify a non default serial id)